nilearn/datasets/description/allen_rsn_2011.rst

```diff
@@ -0,0 +1,29 @@
+Allen 2011
+
+
+Notes
+-----
+Collection of resting-state network templates extracted from 600 healthy
+subjects fMRI data.
+
+
+Content
+-------
+    :'all_unthresh_tmaps': unthresholded t-maps of all 75 components
+    :'rsn_unthresh_tmaps': unthresholded t-maps of 28 resting state networks
+    :'aggregate_ic_comps': the aggregate ICA components
+    :'all_unthresh_labels': labels and indices of the 28 RSN in the 'all_unthresh_tmaps' file
```

Comment:
no, the indices are for the `all_unthresh_tmaps` file. in the `rsn_unthresh_tmaps` file these have a different order and explained in the cited paper. rsn_unthresh_tmaps is a subset of all_unthresh_tmaps. I see the confusion, however this is how the information and files are available from the source.